pages/api/id/[id].js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { ingredientsWithMeasure } from '../../../util/functions';
+
+export default async function getRecipeDetailsById(req, res) {
```

Comment:
I think naming this route something like `/api/recipe/[id]` would make more sense

Reply:
I didn't think of name when I made this file, thanks!